package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "jest",
-    "start": "babel-node index.js"
+    "start": "babel-node index.js",
+    "lint": "./node_modules/.bin/eslint ./src"
```

Comment:
Since the package.json runs files located in node_modules/.bin, i think you could just put `eslint ./src`